Configs/.local/lib/hyde/luautils/selector/rofi.lua

```
@@ -0,0 +1,189 @@
+        string.format(
+        "window{border:%spx;border-radius:%spx;} wallbox{border-radius:%spx;} element{border-radius:%spx;}",
+        hypr.border_size,
+        hypr.win_border,
+        hypr.elem_border,
+        hypr.elem_border
+    )
+    local placeholder_theme = string.format('entry { placeholder: "%s"; }', (placeholder:gsub('"', '\\"')))
+
+    local lines = {}
+    for _, item in ipairs(items) do
+        lines[`#lines` + 1] = (item.icon or "") .. "\t" .. (item.name or "")
```

Luacheck (1.2.0)

[error] expected expression near '`'

(E011)